tests/components/smappee/test_config_flow.py

```diff
@@ -54,3 +222,38 @@ async def test_full_flow(hass, aiohttp_client, aioclient_mock):
 
     assert len(hass.config_entries.async_entries(DOMAIN)) == 1
     assert len(mock_setup.mock_calls) == 1
+
+
+async def test_full_zeroconf_flow(hass):
+    """Test the full zeroconf flow."""
+    flow = config_flow.SmappeeFlowHandler()
```

Comment:
Please don't create the flow like this. Start the test with `hass.config_entries.flow.async_init`.

Reply:
Can you clarify why we can't create the flow like this? The Elgato and WLED tests are quite similar.